ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go

```diff
@@ -162,6 +162,8 @@ spec:
   sshPublicKey: "ssh-rsa "
   clusters:
   - clusterName: "cluster1"
+    apiVIP: 10.16.231.2
```

Comment:
same don't change old unit test data

Reply:
I added this to ensure that both the `apiVIP` and `apiVIPs` syntax are tested. Is this unit test even valid without `apiVIP`? It's a unit test for a standard (multinode) cluster, I believe that apiVIP would be a required field.

Without this, there isn't a test that includes `apiVIP` (that I saw)

Reply:
okay, that is strange it should complain if there is no api or ingress IPs set